pkgs/development/libraries/gettext-runtime/default.nix

```diff
@@ -1,22 +1,18 @@
 { stdenv, lib, fetchurl, libiconv, xz, fetchpatch }:
 
-stdenv.mkDerivation rec {
-  pname = "gettext";
-  version = "0.20.1";
+let common = import ../gettext/common.nix { inherit fetchurl; };
+in stdenv.mkDerivation rec {
+  pname = "gettext-runtime";
+  version = common.version;
+  src = common.src;
+
+  sourceRoot = "${common.pname}-${common.version}/${pname}/";
```

Comment:
```suggestion
  sourceRoot = "${common.src.name}/${pname}/";
```

Reply:
`${common.src.name}` actually resolves to `gettext-0.20.1.tar.gz`, so this is not what we want.